atkinson/config/search.py

```diff
@@ -6,34 +6,43 @@
 import os
 
 
+# Using a tuple here over a list to insure it's immutable.
+DEFAULT_CONFIG_PATHS = (os.path.realpath('./configs'),
```

Comment:
Just curious why you made this a tuple, and not simply a list?

Reply:
I think the comment explains, but a global constant can me modified via the 'global' function. If a tuple is used, you can't simply append more data to this constant since a tuple is immutable. Instead you would have to:
 ```
global DEFAULT_CONFIG_PATHS
DEFAULT_CONFIG_PATH = <new_data>
```
which would be a bigger code smell.

Reply:
Sorry, I totally didn't see the comment above to declaration for some reason.  However, are we really that worried that a user is going to try to mess with this constant?  I guess it doesn't matter that much to me either way, it just seems like extra work to use this when it is a tuple, as you will have to convert it to a list for most use cases.